test/Interop/src/index.js

```diff
@@ -15,13 +15,14 @@ function mkElement(id) {
 }
 
 // Create a valid input
-const items = [ { name: "Thomas" }, { name: "Chris" }, { name: "Qian" }, { name: "Jeff" } ]
+const items = [ { name: "Thomas" }, { name: "Chris" }, { name: "Qian", email: "qian@cn.com" }, { name: "Jeff" } ]
 const input =
   { items: [] // empty to start
   , debounceTime: 300
   , placeholder: "Type to search..."
   , key: "name"
   , keepOpen: false
+  , insertable: true
```

Comment:
@lee00678 with this update you'll have to change the input for _all_ existing configs to include this option

Reply:
@lee00678 one day you'll have to tell me about the origin of this username

Reply:
@thomashoneyman ikr? Always confuses me

Reply:
Actually I just tested this and it looks like it doesn't break whet `insertable` is left out, just treats it as `false`, so I'm going to release this a a minor update

Reply:
> @lee00678 one day you'll have to tell me about the origin of this username

never.